addons/google_calendar/models/google_calendar.py

```diff
@@ -546,14 +538,12 @@ def synchronize_events_cron(self, cr, uid, context=None):
                 _logger.info("[%s] Calendar Synchro - Exception : %s !" % (user_to_sync, exception_to_unicode(e)))
         _logger.info("Calendar Synchro - Ended by cron")
 
-    def synchronize_events(self, cr, uid, ids, lastSync=True, context=None):
-        if context is None:
-            context = {}
+    def synchronize_events(self, lastSync=True):
```

Comment:
I think there is some mess here, the old method was defined with ids, while ids are always None, either this method is going to call through controller or cron, so there will be not id, so there is no need to decorate it through api.multi(Even no need of decoration).
Regarding ids, it is used in below line
```user_to_sync = self.ids and self.ids[0] or self.env.uid```
It is confusing, there will be no ids when why should we ids attribute at all.
I would say user_to_sync should be env.uid and there is no need of ids at all.